'''真核转录'''是[[真核生物|真核生物]]将存储在[[脱氧核糖核酸|DNA]]中的[[遗传信息|遗传信息]]复制到[[核糖核酸|RNA]]中的复杂过程。[[转录|转录]]既发生在真核生物中，也发生在[[原核生物|原核生物]]中。原核生物中的[[RNA聚合酶|RNA聚合酶]]可以启动所有类型的转录；不同于原核生物，真核生物（包括人类）的RNA聚合酶有三种变体，每种变体编码不同类型的[[基因|基因]]。真核生物的转录与[[翻译_(生物学)|翻译]]在不同的场所进行。真核细胞的转录发生在细胞核内，核中的DNA一般被包装成[[核小体|核小体]]以及更高级的[[染色质|染色质]]结构。真核生物基因组的复杂性决定了基因表达控制的多样性和复杂性。

==概述==
转录是将存储在DNA链中的遗传信息复制到mRNA{{tsl|en|Complementarity (molecular biology)|互补链}}中的过程。<ref name=Watson /> 真核转录发生在细胞核中，并在分为三个阶段：起动，延伸和终止。<ref name=Watson>{{cite book|last=Watson|first=James|title=Molecular Biology of the Gene|publisher=Benjamin-Cummings Publishing Company|isbn=978-0-321-76243-6|edition=7th|author2=Tania A. Baker |author3=Stephen P. Bell |author4=Alexander Gann |author5=Michael Levine |author6=Richard Losik |author7=Stephen C. Harrison }}</ref> 催化这一复杂过程的是三种多亚基RNA聚合酶。其中RNA聚合酶I负责转录编码[[核糖体RNA|核糖体RNA]]（rRNA）的基因。
<ref name=Watson />

编码蛋白质d的基因被转录成[[信使RNA|信使RNA]]（mRNA），mRNA将遗传信息从[[细胞核|细胞核]]中的DNA处传递到[[细胞质|细胞质]]中的蛋白质合成位点。<ref name=Watson />尽管mRNA的种类很丰富，但它们并不是细胞中最丰富的RNA种类；所谓的非编码RNA占细胞转录输出的绝大部分。<ref name=Mattick>{{cite journal|last=Mattick|first=J. S.|title=Non-coding RNAs: the architects of eukaryotic complexity|journal=EMBO Reports|date=1 November 2001|volume=2|issue=11|pages=986–991|doi=10.1093/embo-reports/kve230|pmid=11713189|pmc=1084129}}</ref> 这些非编码RNA在细胞中具有多种重要的功能。<ref name=Mattick />

==RNA聚合酶==

[[Image:Alpha-Amanitin–RNA_polymerase_II_complex_1K83.png|thumb]]（又称死亡帽）中发现的一种剧毒物质]]

真核生物有三种核RNA聚合酶，每种都具有不同的作用和特性。<ref name="Lodish 7th">{{cite book|last=Lodish Harvey|first=...[et al]|title=Molecular cell biology|publisher=W.H. Freeman and Co.|location=New York|isbn=9781429234139|edition=7th}}</ref><ref name="Cramer">{{cite journal |vauthors=Cramer P, Armache KJ, Baumli S, Benkert S, Brueckner F, Buchen C, Damsma GE, Dengl S, Geiger SR, Jasiak AJ, Jawhari A, Jennebach S, Kamenski T, Kettenberger H, Kuhn CD, Lehmann E, Leike K, Sydow JF, Vannini A |title=Structure of eukaryotic RNA polymerases |journal=Annu Rev Biophys |volume=37 |issue= |pages=337–52 |year=2008 |pmid=18573085 |doi=10.1146/annurev.biophys.37.032807.130008 |url=}}</ref>

{| class="wikitable"
| '''名称''' || '''场所''' || '''产物'''
 |- 
 | {{tsl|en|RNA polymerase I|RNA聚合酶Ⅰ}} (Pol I, Pol A)  || [[核仁|核仁]]  || 绝大部分的[[核糖體核糖核酸|核糖体RNA]]([[28S_rRNA|28S rRNA]]、{{tsl|en|18S ribosomal RNA|18S rRNA}}、[[5.8S_rRNA|5.8S rRNA]])
 |- 
 | [[RNA聚合酶Ⅱ|RNA聚合酶Ⅱ]] (Pol II, Pol B) || 细胞核 ||[[信使核糖核酸|信使RNA]]（mRNA）、大多数[[小核核糖核酸|小核RNA]]（snRNA）、[[小分子干扰核糖核酸|小分子干扰RNA]]（siRNA）和 [[小分子核糖核酸|微RNA]]（miRNA）。
 |- 
 | {{tsl|en|RNA polymerase III|RNA聚合酶Ⅲ}} (Pol III, Pol C) || 细胞核（可能还有核仁-[[核质|核质]]界面） || [[转运核糖核酸|转运RNA]]（tRNA）、其它小RNA（包括[[5S_核糖体RNA|5S rRNA]]、snRNA U6、{{tsl|en|signal recognition particle RNA|信号识别颗粒RNA}}（SRP RNA）和其它的稳定短链RNA。
|}

RNA聚合酶I（Pol I）可以催化除5S外的所有rRNA基因的转录。<ref name="Lodish 7th" /><ref name=Cramer /> 编码这些rRNA的为同一个基因，该基因表达时会形成连续的RNA转录物，然后该前体RNA会被剪接成3种rRNA：18S、5.8S、和28S。rRNA基因的转录发生在细胞核的特化结构——核仁中。<ref>{{cite journal|last=Sirri|first=Valentina|author2=Silvio Urcuqui-Inchima |author3=Pascal Roussel |author4=Danièle Hernandez-Verdun |title=Nucleolus: the fascinating nuclear body|journal=Histochem Cell Biol.|year=2008|volume=129|issue=1|pages=13–31|doi=10.1007/s00418-007-0359-6|pmc=2137947|pmid=18046571}}</ref> 转录产生的rRNA与[[蛋白质|蛋白质]]结合形成[[核糖体|核糖体]]。<ref>{{cite journal|last=Fromont-Racine|first=Micheline|author2=Senger, Bruno |author3=Saveanu, Cosmin |author4= Fasiolo, Franco |title=Ribosome assembly in eukaryotes|journal=Gene|date=August 2003|volume=313|pages=17–42|doi=10.1016/S0378-1119(03)00629-2}}</ref>

RNA聚合酶II（Pol II）负责所有mRNA、siRNA、多数snRNA和所有miRNA的转录。<ref name="Lodish 7th" /><ref name=Cramer /> Pol II的转录产物多为单链的{{tsl|en|Primary transcript|前体mRNA}}（hnRNA），这些hnRNA会被进一步加工为成熟的mRNA。<ref name=Watson /> 绝大部分的前体mRNA在通过[[核孔|核孔]]离开细胞核，进入[[细胞质|细胞质]]进行蛋白质的翻译之前就已被加工完成。

RNA聚合酶III（Pol III）负责转录小的非编码RNA，包括tRNA，5S rRNA，U6 snRNA，SRP RNA和其他稳定的短RNA，例如[[核糖核酸酶P|核糖核酸酶P]] RNA.<ref>{{cite journal|last=Dieci|first=Giorgio|author2=Fiorino, Gloria |author3=Castelnuovo, Manuele |author4=Teichmann, Martin |author5= Pagano, Aldo |title=The expanding RNA polymerase III transcriptome|journal=Trends in Genetics|date=December 2007|volume=23|issue=12|pages=614–622|doi=10.1016/j.tig.2007.09.001|pmid=17977614}}</ref>

<!--
RNA Polymerases I, II, and III contain 14, 12, and 17 subunits, respectively.<ref name=Carter>{{cite journal|last=Carter|first=R.|author2=Drouin, G.|title=The Increase in the Number of Subunits in Eukaryotic RNA Polymerase III Relative to RNA Polymerase II Is due to the Permanent Recruitment of General Transcription Factors|journal=Molecular Biology and Evolution|date=21 December 2009|volume=27|issue=5|pages=1035–1043|doi=10.1093/molbev/msp316|pmid=20026480}}</ref> All three eukaryotic polymerases have five core subunits that exhibit homology with the β, β’, α<sup>I</sup>, α<sup>II</sup>, and ω subunits of E. coli RNA polymerase. An identical ω-like subunit (RBP6) is used by all three eukaryotic polymerases, while the same α-like subunits are used by Pol I and III. The three eukaryotic polymerases share four other common subunits among themselves. The remaining subunits are unique to each RNA polymerase. The additional subunits found in Pol I and Pol III relative to Pol II, are homologous to Pol II transcription factors.<ref name=Carter />

Crystal structures of RNA polymerases I<ref>{{cite journal|last=Fernández-Tornero|first=Carlos|author2=Moreno-Morcillo, María |author3=Rashid, Umar J. |author4=Taylor, Nicholas M. I. |author5=Ruiz, Federico M. |author6=Gruene, Tim |author7=Legrand, Pierre |author8=Steuerwald, Ulrich |author9= Müller, Christoph W.  |title=Crystal structure of the 14-subunit RNA polymerase I|journal=Nature|date=23 October 2013|volume=502|issue=7473|pages=644–649|doi=10.1038/nature12636}}</ref> and II<ref>{{cite journal|last=Cramer|first=P.|title=Structural Basis of Transcription: RNA Polymerase II at 2.8 Angstrom Resolution|journal=Science|date=19 April 2001|volume=292|issue=5523|pages=1863–1876|doi=10.1126/science.1059493|pmid=11313498}}</ref> provide an opportunity to understand the interactions among the subunits and the molecular mechanism of eukaryotic transcription in atomic detail.

The [[C端|C端]] domain (CTD) of {{tsl|en|POLR2A}}, the largest subunit of RNA polymerase II, plays an important role in bringing together the machinery necessary for the synthesis and processing of Pol II transcripts.<ref name=corden>{{cite journal|last=Corden|first=Jeffry L.|title=RNA Polymerase II C-Terminal Domain: Tethering Transcription to Transcript and Template|journal=Chemical Reviews|date=13 November 2013|volume=113|issue=11|pages=8423–8455|doi=10.1021/cr400158h|pmid=24040939|pmc=3988834}}</ref> Long and structurally disordered, the CTD contains multiple repeats of heptapeptide sequence YSPTSPS that are subject to [[磷酸化|磷酸化]] and other [[翻译后修饰|翻译后修饰]]s during the transcription cycle. These modifications and their regulation constitute the operational code for the CTD to control transcription initiation, elongation and termination and to couple transcription and RNA processing.<ref name=corden />

==Initiation==

The initiation of gene transcription in eukaryotes occurs in specific steps.<ref name=Watson />  First, an RNA polymerase along with general [[转录因子|转录因子]]s binds to the [[啟動子|啟動子]] region of the gene to form a closed complex called the {{tsl|en|Transcription preinitiation complex}}. The subsequent transition of the complex from the closed state to the open state results in the melting or separation of the two DNA strands and the positioning of the template strand to the active site of the RNA polymerase. Without the need of a primer, RNA polymerase can initiate the synthesis of a new RNA chain using the template DNA strand to guide ribonucleotide selection and polymerization chemistry.<ref name=Watson /> However, many of the initiated syntheses are aborted before the transcripts reach a significant length (~10 nucleotides). During these abortive cycles, the polymerase keeps making and releasing short transcripts until it is able to produce a transcript that surpasses ten nucleotides in length. Once this threshold is attained, RNA polymerase passes the promoter and transcription proceeds to the elongation phase.<ref name=Watson />

[[Image:Simple_transcription_initiation1.svg|thumb]].]]

===Eukaryotic promoters and general transcription factors===
Pol II-transcribed genes contain a region in the immediate vicinity of the transcription start site (TSS) that binds and positions the preinitiation complex. This region is called the core promoter because of its essential role in transcription initiation.<ref>{{cite journal|last=Butler|first=Jennifer|author2=James T. Kadonaga|title=The RNA polymerase II core promoter: a key component in the regulation of gene expression|journal=Genes Dev.|year=2002|volume=16|issue=20|pages=2583–2592|doi=10.1101/gad.1026202|pmid=12381658}}</ref><ref>{{cite journal|last=Lenhard|first=Boris|author2=Sandelin, Albin |author3=Carninci, Piero |title=Metazoan promoters: emerging characteristics and insights into transcriptional regulation|journal=Nature Reviews Genetics|date=6 March 2012|doi=10.1038/nrg3163|pmid=22392219|volume=13|pages=233–45}}</ref> Different classes of sequence elements are found in the promoters. For example, the [[TATA盒|TATA盒]] is the highly conserved DNA recognition sequence for the TATA box binding protein, [[TATA结合蛋白|TATA结合蛋白]], whose binding initiates transcription complex assembly at many genes.

Eukaryotic genes also contain regulatory sequences beyond the core promoter. These [[顺式作用元件|顺式作用元件]] bind transcriptional activators or repressors to increase or decrease transcription from the core promoter. Well-characterized regulatory elements include [[強化子|強化子]], [[沉默子|沉默子]], and [[绝缘子_(基因)|绝缘子 (基因)]]. These regulatory sequences can be spread over a large genomic distance, sometimes located hundreds of kilobases from the core promoters.<ref name=Watson />

General transcription factors are a group of proteins involved in transcription initiation and regulation.<ref name=Watson /> These factors typically have DNA-binding domains that bind specific sequence elements of the core promoter and help recruit RNA polymerase to the transcriptional start site.
General transcription factors for RNA polymerase II include {{tsl|en|Transcription factor II D}}, {{tsl|en|Transcription factor II A}}, {{tsl|en|Transcription factor II B}}, {{tsl|en|Transcription factor II F}}, {{tsl|en|Transcription factor II E}}, and {{tsl|en|Transcription factor II H}}.<ref name=Watson /><ref>{{cite journal|last=Orphanides|first=G|author2=Lagrange, T |author3=Reinberg, D |title=The general transcription factors of RNA polymerase II.|journal=Genes & Development|date=1 November 1996|volume=10|issue=21|pages=2657–2683|doi=10.1101/gad.10.21.2657|pmid=8946909}}</ref><ref>{{cite journal|last=ROEDER|first=R|title=The role of general initiation factors in transcription by RNA polymerase II|journal=Trends in Biochemical Sciences|date=September 1996|volume=21|issue=9|pages=327–335|doi=10.1016/S0968-0004(96)10050-5}}</ref>

===Assembly of preinitiation complex===
To prepare for transcription, a complete set of general transcription factors and RNA polymerase need to be assembled at the core promoter to form the ~2 million dalton preinitiation complex.<ref>{{cite journal|last=He|first=Yuan|author2=Jie Fang |author3=Dylan J. Taatjes |author4=Eva Nogales |title=Structural visualization of key steps in human transcription initiation|journal=Nature|date=28 March 2013|volume=495|pages=481–486|doi=10.1038/nature11991 |pmid=23446344 |pmc=3612373}}</ref>  For example, for promoters that contain a TATA box near the TSS, the recognition of TATA box by the TBP subunit of TFIID initiates the assembly of a transcription complex. The next proteins to enter are TFIIA and TFIIB, which stabilize the DNA-TFIID complex and recruit Pol II in association with TFIIF and additional transcription factors. TFIIF serves as the bridge between the TATA-bound TBP and polymerase. One of the last transcription factors to be recruited to the preinitiation complex is TFIIH, which plays an important role in promoter melting and escape.<ref name=holstege />

[[Image:Preinitiation_complex.png|thumb]].]]

===Promoter melting and open complex formation===
For pol II-transcribed genes, and unlike bacterial RNA polymerase, promoter melting requires hydrolysis of ATP and is mediated by TFIIH.<ref name=holstege>{{cite journal|last=Holstege|first=F. C.P.|title=Three transitions in the RNA polymerase II transcription complex during initiation|journal=The EMBO Journal|date=15 December 1997|volume=16|issue=24|pages=7468–7480|doi=10.1093/emboj/16.24.7468|pmid=9405375|pmc=1170346}}</ref>  TFIIH is a ten-subunit protein, including both [[ATP酶|ATP酶]] and [[蛋白激酶|蛋白激酶]] activities.<ref>{{cite journal|last=SVEJSTRUP|first=J|author2=VICHI, P |author3=EGLY, J |title=The multiple roles of transcription/repair factor TFIIH|journal=Trends in Biochemical Sciences|date=September 1996|volume=21|issue=9|pages=346–350|doi=10.1016/S0968-0004(96)10046-3|pmid=8870499}}</ref> While the upstream promoter DNA is held in a fixed position by TFIID, TFIIH pulls downstream double-stranded DNA into the cleft of the polymerase, driving the separation of DNA strands and the transition of the preinitiation complex from the closed to open state. TFIIB aids in open complex formation by binding the melted DNA and stabilizing the [[transcription bubble
]].

===Abortive initiation===
{{Main|Abortive initiation}}
Once the initiation complex is open, the first ribonucleotide is brought into the active site to initiate the polymerization reaction in the absence of a primer.<ref name=Watson /> This generates a nascent RNA chain that forms a hetero-duplex with the template DNA strand. However, before entering the elongation phase, polymerase may terminate prematurely and release a short, truncated transcript. This process is called abortive initiation.<ref name=revyakin /> Many cycles of abortive initiation may occur before the transcript grows to sufficient length to promote polymerase escape from the promoter. Throughout abortive initiation cycles, RNA polymerase remains bound to the promoter and pulls downstream DNA into its catalytic cleft in a scrunching-kind of motion.<ref name=revyakin>{{cite journal|vauthors=Revyakin A, Liu C, Ebright RH, Strick TR |title=Abortive Initiation and Productive Initiation by RNA Polymerase Involve DNA Scrunching|journal=Science|date=17 November 2006|volume=314|issue=5802|pages=1139–1143|doi=10.1126/science.1131398|pmid=17110577|pmc=2754787}}</ref>

===Promoter escape===
When a transcript attains the threshold length of ten nucleotides, it enters the RNA exit channel.<ref name=Watson /> The polymerase breaks its interactions with the promoter elements and any regulatory proteins associated with the initiation complex that it no longer needs.<ref>{{cite journal|last=Dvir|first=Arik|title=Promoter escape by RNA polymerase II|journal=Biochimica et Biophysica Acta (BBA) - Gene Structure and Expression|date=September 2002|volume=1577|issue=2|pages=208–223|doi=10.1016/S0167-4781(02)00453-0|pmid=12213653}}</ref> Promoter escape in eukaryotes requires ATP hydrolysis and, in the case of Pol II-phosphorylation of the CTD. Meanwhile, the transcription bubble collapses down to 12-14 nucleotides, providing kinetic energy required for the escape.<ref name=Watson />

==Elongation==
After escaping the promoter and shedding most of the transcription factors for initiation, the polymerase acquires new factors for the next phase of transcription: elongation.<ref>{{cite journal|last=Pokholok|first=Dmitry K|author2=Hannett, Nancy M |author3=Young, Richard A |title=Exchange of RNA Polymerase II Initiation and Elongation Factors during Gene Expression In Vivo|journal=Molecular Cell|date=April 2002|volume=9|issue=4|pages=799–809|doi=10.1016/S1097-2765(02)00502-6}}</ref><ref>{{cite journal|last=Wade|first=Joseph T|author2=Struhl, Kevin|title=The transition from transcriptional initiation to elongation|journal=Current Opinion in Genetics & Development|date=April 2008|volume=18|issue=2|pages=130–136|doi=10.1016/j.gde.2007.12.008|pmc=2563432}}</ref> Transcription elongation is a {{tsl|en|Processivity}} process. Double stranded DNA that enters from the front of the enzyme is unzipped to avail the template strand for RNA synthesis. For every DNA [[碱基对|碱基对]] separated by the advancing polymerase, one hybrid RNA:DNA base pair is immediately formed. DNA strands and nascent RNA chain exit from separate channels; the two DNA strands reunite at the trailing end of the transcription bubble while the single strand RNA emerges alone.

===Elongation factors===
Among the proteins recruited to polymerase are elongation factors, thus called because they stimulate transcription elongation.<ref>{{cite journal|last=Saunders|first=Abbie|author2=Core, Leighton J. |author3=Lis, John T. |title=Breaking barriers to transcription elongation|journal=Nature Reviews Molecular Cell Biology|date=August 2006|volume=7|issue=8|pages=557–567|doi=10.1038/nrm1981|pmid=16936696}}</ref> There are different classes of elongation factors. Some factors can increase the overall rate of transcribing, some can help the polymerase through transient pausing sites, and some can assist the polymerase to transcribe through chromatin.<ref>{{cite journal|last=Arndt|first=Karen|author2=Caroline Kane|title=Running with RNA polymerase: eukaryotic transcript elongation|journal=Trends in Genetics|date=1 October 2003|volume=19|issue=10|pages=543–550|doi=10.1016/j.tig.2003.08.008|pmid=14550628}}</ref> One of the elongation factors, {{tsl|en|P-TEFb}}, is particularly important.<ref name=bres>{{cite journal|last=Brès|first=Vanessa|author2=Yoh, Sunnie M |author3=Jones, Katherine A |title=The multi-tasking P-TEFb complex|journal=Current Opinion in Cell Biology|date=June 2008|volume=20|issue=3|pages=334–340|doi=10.1016/j.ceb.2008.04.008|pmc=2628440}}</ref> P-TEFb phosphorylates the second residue (Ser-2) of the CTD repeats (YSPTSPS) of the bound Pol II. P-TEFb also phosphorylates and activates SPT5 and TAT-SF1. SPT5 is a universal transcription factor that helps recruit [[5′端帽|5′端帽]] enzyme to Pol II with a CTD phosphorylated at Ser-5. TAF-SF1 recruits components of the RNA splicing machinery to the Ser-2 phosphorylated CTD. P-TEFb also helps suppress transient pausing of polymerase when it encounters certain sequences immediately following initiation.<ref name=bres />

===Transcription fidelity===
Transcription fidelity is achieved through multiple mechanisms. RNA polymerases select correct [[核苷三磷酸|核苷三磷酸]] (NTP) substrate to prevent transcription errors. Only the NTP which correctly base pairs with the coding base in the DNA is admitted to the active center.<ref>{{cite journal|last=Westover|first=Kenneth D.|author2=Bushnell, David A. |author3=Kornberg, Roger D. |title=Structural Basis of Transcription|journal=Cell|date=November 2004|volume=119|issue=4|pages=481–489|doi=10.1016/j.cell.2004.10.016}}</ref><ref>{{cite journal|last=Wang|first=Dong|author2=Bushnell, David A. |author3=Westover, Kenneth D. |author4=Kaplan, Craig D. |author5= Kornberg, Roger D. |title=Structural Basis of Transcription: Role of the Trigger Loop in Substrate Specificity and Catalysis|journal=Cell|date=December 2006|volume=127|issue=5|pages=941–954|doi=10.1016/j.cell.2006.11.023 |pmid=17129781 |pmc=1876690}}</ref> RNA polymerase performs two known proof reading functions to detect and remove misincorporated nucleotides: pyrophosphorylytic editing and hydrolytic editing.<ref name=Watson /> The former removes the incorrectly inserted ribonucleotide by a simple reversal of the polymerization reaction, while the latter involves backtracking of the polymerase and cleaving of a segment of error-containing RNA product. Elongation factor TFIIS stimulates an inherent ribonuclease activity in the polymerase, allowing the removal of misincorporated bases through limited local RNA degradation.<ref>{{cite journal|last=Wang|first=D.|author2=Bushnell, D. A. |author3=Huang, X. |author4=Westover, K. D. |author5=Levitt, M. |author6= Kornberg, R. D. |title=Structural Basis of Transcription: Backtracked RNA Polymerase II at 3.4 Angstrom Resolution|journal=Science|date=28 May 2009|volume=324|issue=5931|pages=1203–1206|doi=10.1126/science.1168729 |pmid=19478184 |pmc=2718261}}</ref> Note that all reactions (phosphodiester bond synthesis, pyrophosphorolysis, phosphodiester bond hydrolysis) are performed by RNA polymerase by using a single active center.<ref>{{cite journal|last=Sosunov|first=Vasily|author2=Ekaterina Sosunova |author3=Arkady Mustaev |author4=Irina Bass |author5=Vadim Nikiforov |author6=Alex Goldfarb |title=Unified two-metal mechanism of RNA synthesis and degradation by RNA polymerase|journal=The EMBO Journal|year=2003|volume=22|pages=2234–2244|doi=10.1093/emboj/cdg193 |pmid=12727889 |pmc=156065}}</ref>

===Pausing, poising, and backtracking===
Transcription elongation is not a smooth ride along the DNA railway. For proofreading, the polymerase is made to back-up, erase some of the RNA it has already made and have another go at transcription.<ref name=Watson /> In general, RNA polymerase does not transcribe through a gene at a constant pace. Rather it pauses periodically at certain sequences, sometimes for long periods of time before resuming transcription.<ref name=Adelman /> In extreme cases, for example, when the polymerase encounters a damaged nucleotide, it comes to a complete halt. More often, an elongating polymerase is stalled near the promoter.<ref name=Adelman>{{cite journal|last=Adelman|first=Karen|author2=Lis, John T.|title=Promoter-proximal pausing of RNA polymerase II: emerging roles in metazoans|journal=Nature Reviews Genetics|date=18 September 2012|volume=13|issue=10|pages=720–731|doi=10.1038/nrg3293|pmc=3552498}}</ref> Promoter-proximal pausing during early elongation is a commonly used mechanism for regulating genes poised to be expressed rapidly or in a coordinated fashion. Pausing is mediated by a complex called NELF (negative elongation factor) in collaboration with DSIF (DRB-sensitivity-inducing factor containing SPT4/SPT5).<ref>{{cite journal|last=Missra|first=A.|author2=Gilmour, D. S.|title=Interactions between DSIF (DRB sensitivity inducing factor), NELF (negative elongation factor), and the Drosophila RNA polymerase II transcription elongation complex|journal=Proceedings of the National Academy of Sciences|date=4 June 2010|volume=107|issue=25|pages=11301–11306|doi=10.1073/pnas.1000681107|pmid=20534440|pmc=2895096}}</ref> The blockage is released once the polymerase receives an activation signal, such as the phosphorylation of Ser-2 of CTD tail by P-TEFb. Other elongation factors such as ELL and TFIIS stimulate the rate of elongation by limiting the length of time that polymerase pauses.<ref name=Watson />

===RNA processing===
Elongating polymerase is associated with a set of protein factors required for various types of RNA processing.<ref name=Watson /> mRNA is capped as soon as it emerges from the RNA-exit channel of the polymerase. After capping, dephosphorylation of Ser-5 within the CTD repeats may be responsible for dissociation of the capping machinery. Further phosphorylation of Ser-2 causes recruitment of the RNA splicing machinery that catalyzes the removal of non-coding introns to generate mature mRNA.<ref name=Watson /> Alternative splicing expands the protein complements in eukaryotes. Just as with 5’-capping and splicing, the CTD tail is involved in recruiting enzymes responsible for [[多腺苷酸化|多腺苷酸化]], the final RNA processing event that is coupled with the termination of transcription.<ref name=Watson />

==Termination==
The last stage of transcription is termination, which leads to the dissociation of the complete transcript and the release of RNA polymerase from the template DNA.The process differs for each of the three RNA polymerases.<ref name="Richard_rnap"/> 
The mechanism of termination is the least understood of the three transcription stages.

===Factor-dependent===
The termination of transcription of pre-rRNA genes by polymerase Pol I is performed by a system that needs a specific transcription termination factor.<ref name="Lodish 7th" /> The mechanism used bears some resemblance to the rho-dependent termination in prokaryotes.<ref name=Suz-nature>{{cite journal|last=Clancey|first=Suzanne|title=DNA transcription|journal=Nature Education|year=2008|volume=1|issue=41|url=http://www.nature.com/scitable/topicpage/dna-transcription-426|accessdate=27 November 2013}}</ref> Eukaryotic cells contain hundreds of ribosomal DNA repeats, sometimes distributed over multiple chromosomes. Termination of transcription occurs in the ribosomal intergenic spacer region that contains several transcription termination sites upstream of a Pol I pausing site. Through a yet unknown mechanism, the 3’-end of the transcript is cleaved, generating a large primary rRNA molecule that is further processed into the mature 18S, 5.8S and 28S rRNAs.

As Pol II reaches the end of a gene, two protein complexes carried by the CTD, CPSF (cleavage and polyadenylation specificity factor) and CSTF (cleavage stimulation factor), recognize the poly-A signal in the transcribed RNA.<ref name=Richard_rnap /> Poly-A-bound CPSF and CSTF recruit other proteins to carry out RNA cleavage and then polyadenylation. Poly-A polymerase adds approximately 200 adenines to the cleaved 3’ end of the RNA without a template.<ref name=Richard_rnap /> The long poly-A tail is unique to transcripts made by Pol II.<br />
 
In the process of terminating transcription by Pol I and Pol II, the elongation complex does not dissolve immediately after the RNA is cleaved. The polymerase continues to move along the template, generating a second RNA molecule associated with the elongation complex.<ref name=Watson /> Two models have been proposed to explain how termination is achieved at last.<ref name=Richard_rnap>{{cite journal|last=Richard|first=P.|author2=Manley, J. L.|title=Transcription termination by nuclear RNA polymerases|journal=Genes & Development|date=1 June 2009|volume=23|issue=11|pages=1247–1269|doi=10.1101/gad.1792809|pmc=2763537}}</ref>  The allosteric model states that when transcription proceeds through the termination sequence, it causes disassembly of elongation factors and/or an assembly of termination factors that cause conformational changes of the elongation complex.<ref name=Suz-nature /><ref name=RNAPII>{{cite journal|last=Rosonina|first=E.|title=Terminating the transcript: breaking up is hard to do|journal=Genes & Development|date=1 May 2006|volume=20|issue=9|pages=1050–1056|doi=10.1101/gad.1431606|pmid=16651651}}</ref>  The torpedo model suggests that a 5' to 3' [[外切酶|外切酶]] degrades the second RNA as it emerges from the elongation complex. Polymerase is released as the highly processive exonuclease overtakes it. It is proposed that an emerging view will express a merge of these two models.<ref name=RNAPII />

===Factor-independent===
RNA polymerase III can terminate transcription efficiently without the involvement of additional factors. The Pol III termination signal consists of a stretch of [[胸腺嘧啶|胸腺嘧啶]]s (on the nontemplate strand) located within 40bp downstream from the 3' end of mature RNAs.<ref name=Richard_rnap /> The poly-T termination signal pauses Pol III and causes it to backtrack to the nearest [[莖環|莖環]] to become a “dead-end” complex.<ref>{{cite journal|last=Nielsen|first=S.|author2=Yuzenkova, Y. |author3=Zenkin, N. |title=Mechanism of Eukaryotic RNA Polymerase III Transcription Termination|journal=Science|date=27 June 2013|volume=340|issue=6140|pages=1577–1580|doi=10.1126/science.1237934|pmc=3760304}}</ref> Consistent with the allosteric mechanism of termination,<ref>{{cite journal|last=Epshtein|first=Vitaly|author2=Cardinale, Christopher J. |author3=Ruckenstein, Andrei E. |author4=Borukhov, Sergei |author5= Nudler, Evgeny |title=An Allosteric Path to Transcription Termination|journal=Molecular Cell|date=December 2007|volume=28|issue=6|pages=991–1001|doi=10.1016/j.molcel.2007.10.011}}</ref> the RNA hairpin allosterically opens Pol III and causes the elongation complex to disintegrate. The extensive structure embedded in the Pol III-transcript thus is responsible for the factor-independent release of Pol III at the end of a gene. RNA-duplex-dependent termination is an ancient mechanism that dates back to the last universal common ancestor.

==Eukaryotic transcriptional control==
The {{tsl|en|regulation of gene expression}} in eukaryotes is achieved through the interaction of several levels of control that acts both locally to turn on or off individual genes in response to a specific cellular need and globally to maintain a chromatin-wide gene expression pattern that shapes cell identity.<ref name=Watson /><ref>{{cite journal|last=Shandilya|first=J|author2=Robert SG|title=The transcription cycle in eukaryotes: from productive initiation to RNA polymerase II recycling|journal=Biochim Biophys Acta|year=2012|volume=1819|issue=5|pages=391–400|doi=10.1016/j.bbagrm.2012.01.010|pmid=22306664}}</ref> Because eukaryotic genome is wrapped around [[組織蛋白|組織蛋白]]s to form nucleosomes and higher-order chromatin structures, the substrates for transcriptional machinery are in general partially concealed.<ref name=Watson /> Without regulatory proteins, many genes are expressed at low level or not expressed at all. Transcription requires displacement of the positioned nucleosomes to enable the transcriptional machinery to gain access of the DNA.<ref>{{cite journal|last=Kulaeva|first=Olga|author2=Daria Gaykalova |author3=Vasily M. Studitsky |title=Transcription Through Chromatin by RNA polymerase II: Histone Displacement and Exchange|journal=Mutat. Res.|year=2007|volume=618|issue=1–2|pages=116–129|doi=10.1016/j.mrfmmm.2006.05.040|pmc=1924643|pmid=17313961}}</ref>

All steps in the transcription are subject to some degree of regulation.<ref name=Watson /> Transcription initiation in particular is the primary level at which gene expression is regulated. Targeting the rate-limiting initial step is the most efficient in terms of energy costs for the cell. Transcription initiation is regulated by cis-acting elements (enhancers, silencers, isolators) within the regulatory regions of the DNA, and sequence-specific [[反式作用因子|反式作用因子]]s that act as activators or repressors.<ref name=Watson /> Gene transcription can also be regulated post-initiation by targeting the movement of the elongating polymerase.<ref name=Peterlin>{{cite journal|last=Peterlin|first=BM|author2=DH Price|title=Controlling the elongation phase of transcription with P-TEFb|journal=Molecular Cell|year=2006|volume=23|issue=3|pages=297–305|doi=10.1016/j.molcel.2006.06.014|pmid=16885020}}</ref>

===Global control and epigenetic regulation===
{{main|Epigenetic regulation}}

The eukaryotic genome is organized into a compact chromatin structure that allows only regulated access to DNA. The chromatin structure can be globally "open" and more transcriptionally permissive, or globally "condensed" and transcriptionally inactive. The former ([[真染色質|真染色質]]) is lightly packed and rich in genes under active transcription. The latter ([[異染色質|異染色質]]) includes gene-poor regions such as [[端粒|端粒]]s and [[着絲粒|着絲粒]]s but also regions  with normal gene density but transcriptionally silenced. Transcription can be silenced by [[組織蛋白|組織蛋白]] ([[组蛋白脱乙酰酶|组蛋白脱乙酰酶]] and {{tsl|en|Histone methylation}}), [[RNA干扰|RNA干扰]], and/or [[DNA甲基化|DNA甲基化]].<ref>{{cite journal |last=Bannister|first=Andrew J. |author2=Kouzarides, Tony |title=Regulation of chromatin by histone modifications |journal=Cell Research |date=15 February 2011 |volume=21 |issue=3 |pages=381–395 |doi=10.1038/cr.2011.22 |pmid=21321607 |pmc=3193420}}</ref>

The gene expression patterns that define cell identity are inherited through cell division.<ref name=Watson /> This process is called [[表觀遺傳學|表觀遺傳學]]. DNA methylation is reliably inherited through the action of maintenance methylases that modify the nascent DNA strand generated by replication.<ref name=Watson /> In mammalian cells, DNA methylation is the primary marker of transcriptionally silenced regions. Specialized proteins can recognize the marker and recruit [[组蛋白脱乙酰酶|组蛋白脱乙酰酶]] and methylases to re-establish the silencing. Nucleosome histone modifications could also be inherited during cell division, however, it is not clear whether it can work independently without the direction by DNA methylation.<ref name=Watson />

===Gene-specific activation===
The two main tasks of transcription initiation are to provide RNA polymerase with an access to the promoter and to assemble general transcription factors with polymerase into a transcription initiation complex. Diverse mechanisms of initiating transcription by overriding inhibitory signals at the gene promoter have been identified.<ref name=Watson /> Eukaryotic genes have acquired extensive regulatory sequences that encompass a large number of regulator-binding sites and spread overall kilobases (sometimes hundreds of kilobases) from the promoter–-both upstream and downstream.<ref name=Watson /> The regulator binding sites are often clustered together into units called enhancers. Enhancers can facilitate highly cooperative action of several transcription factors (which constitute {{tsl|en|enhanceosome}}s). Remote enhancers allow transcription regulation at a distance. Insulators situated between enhancers and promoters help define the genes that an enhancer can or cannot influence.

Eukaryotic transcriptional activators have separate DNA-binding and activating functions.<ref name=Watson /> Upon binding to its cis-element, an activator can recruit polymerase directly or recruit other factors needed by the transcriptional machinery. An activator can also recruit nucleosome modifiers that alter chromatin in the vicinity of the promoter and thereby help initiation. Multiple activators can work together, either by recruiting a common or two mutually dependent components of the transcriptional machinery, or by helping each other bind to their DNA sites.<ref name=Watson /> These interactions can synergize multiple signaling inputs and produce intricate transcriptional responses to address cellular needs.

===Gene-specific repression ===
Eukaryotic transcription repressors share some of the mechanisms used by their prokaryotic counterparts. For example, by binding to a site on DNA that overlaps with the binding site of an activator, a repressor can inhibit binding of the activator. But more frequently, eukaryotic repressors inhibit the function of an activator by masking its activating domain, preventing its nuclear localization, promoting its degradation, or inactivating it through chemical modifications.<ref name=Watson /> Repressors can directly inhibit transcription initiation by binding to a site upstream of a promoter and interacting with the transcriptional machinery. Repressors can indirectly repress transcription by recruiting histone modifiers (deacetylases and methylases) or nucleosome remodeling enzymes that affect the accessibility of the DNA.<ref name=Watson /> Repressing histone and DNA modifications are also the basis of transcriptional silencing that can spread along the chromatin and switch off multiple genes.<ref name=Brownbook>{{cite book|last=Brown|first=Terrance A.|title=Genomes|year=2002|publisher=Wiley-Liss.|location=New York|isbn=0-471-25046-5}}</ref>

===Elongation and termination control===
The elongation phase starts once assembly of the elongation complex has been completed, and progresses until a termination sequence is encountered.<ref name=Watson /> The post-initiation movement of RNA polymerase is the target of another class of important regulatory mechanisms. For example, the transcriptional activator {{tsl|en|Tat (HIV)}} affects elongation rather than initiation during its regulation of [[人類免疫缺陷病毒|人類免疫缺陷病毒]] transcription.<ref>{{cite journal|last=Kao|first=Shaw-Yi|author2=Andrew Calman |author3=Paul Luciw |author4=Matija Peterlin |title=Anti-termination of transcription within the long terminal repeat of HIV-1 by tat gene product|journal=Nature|date=3 December 1987|volume=330|pages=489–493|doi=10.1038/330489a0|issue=6147 |pmid=2825027}}</ref> In fact, many eukaryotic genes are regulated by releasing a block to transcription elongation called promoter-proximal pausing.<ref>{{cite journal|last=Lis|first=J|title=Promoter-associated Pausing in Promoter Architecture and Postinitiation Transcriptional Regulation|journal=Cold Spring Harb Symp Quant Biol|year=1998|volume=63|pages=347–56|doi=10.1101/sqb.1998.63.347|pmid=10384299}}</ref> Pausing can influence chromatin structure at promoters to facilitate gene activity and lead to rapid or synchronous transcriptional responses when cells are exposed to an activation signal.<ref name=Adelman /> Pausing is associated with the binding of two negative elongation factors, DSIF (SPT4/SPT5) and NELF, to the elongation complex. Other factors can also influence the stability and duration of the paused polymerase.<ref>{{cite journal|vauthors=Cheng B, Li T, Rahl PB, Adamson TE, Loudas NB, Guo J, Varzavand K, Cooper JJ, Hu X, Gnatt A, Young RA, Price DH |title=Functional Association of Gdown1 with RNA Polymerase II Poised on Human Genes|journal=Molecular Cell|date=January 2012|volume=45|issue=1|pages=38–50|doi=10.1016/j.molcel.2011.10.022|pmid=22244331|pmc=3259526}}</ref> Pause release is triggered by the recruitment of the P-TEFb kinase.<ref name=Peterlin />

Transcription termination has also emerged as an important area of transcriptional regulation. Termination is coupled with the efficient recycling of polymerase.<ref>{{cite journal|last=Feige|first=MJ|author2=LM Hendershot|title=Disulfide bonds in ER protein folding and homeostasis|journal=Curr Opin Cell Biol|date=April 2011|volume=23|issue=2|pages=167–75|doi=10.1016/j.ceb.2010.10.012|pmid=21144725|pmc=3078216}}</ref> The factors associated with transcription termination can also mediate gene looping and thereby determine the efficiency of re-initiation.

==Transcription-coupled DNA repair==
When transcription is arrested by the presence of a {{tsl|en|Molecular lesion}} in the transcribed strand of a gene, [[DNA修復|DNA修復]] proteins are recruited to the stalled RNA polymerase to initiate a process called transcription-coupled repair.<ref>{{cite journal|last=MELLON|first=I|title=Selective removal of transcription-blocking DNA damage from the transcribed strand of the mammalian DHFR gene|journal=Cell|date=October 1987|volume=51|issue=2|pages=241–249|doi=10.1016/0092-8674(87)90151-6}}</ref> Central to this process is the general transcription factor TFIIH that has ATPase activity. TFIIH causes a conformational change in the polymerase, to expose the transcription bubble trapped inside, in order for the DNA repair enzymes to gain access to the lesion.<ref>{{cite journal|last=Sarker|first=Altaf H.|author2=Tsutakawa, Susan E. |author3=Kostek, Seth |author4=Ng, Cliff |author5=Shin, David S. |author6=Peris, Marian |author7=Campeau, Eric |author8=Tainer, John A. |author9=Nogales, Eva |author10= Cooper, Priscilla K. |title=Recognition of RNA Polymerase II and Transcription Bubbles by XPG, CSB, and TFIIH: Insights for Transcription-Coupled Repair and Cockayne Syndrome|journal=Molecular Cell|date=October 2005|volume=20|issue=2|pages=187–198|doi=10.1016/j.molcel.2005.09.022|pmid=16246722}}</ref> Thus, RNA polymerase serves as damage-sensing protein in the cell to target repair enzymes to genes that are being actively transcribed.

==Comparisons between prokaryotic and eukaryotic transcription==
Eukaryotic transcription is more complex than {{tsl|en|Bacterial transcription}}. For instance, in eukaryotes the genetic material (DNA), and therefore transcription, is primarily localized to the nucleus, where it is separated from the cytoplasm (in which translation occurs) by the nuclear membrane. This allows for the temporal regulation of gene expression through the sequestration of the RNA in the nucleus, and allows for selective transport of mature RNAs to the cytoplasm. Bacteria do not have a distinct nucleus that separates DNA from ribosome and mRNA is translated into protein as soon as it is transcribed. The coupling between the two processes provides an important mechanism for prokaryotic gene regulation.<ref name=Watson />

At the level of initiation, RNA polymerase in prokaryotes (bacteria in particular) binds strongly to the promoter region and initiates a high basal rate of transcription. No ATP hydrolysis is needed for the close-to-open transition, promoter melting is driven by binding reactions that favor the melted conformation. Chromatin greatly impedes transcription in eukaryotes. Assembly of large multi-protein preinitiation complex is required for promoter-specific initiation. Promoter melting in eukaryotes requires hydrolysis of ATP. As a result, eukaryotic RNA polymerases exhibit a low basal rate of transcription initiation.<ref name=Brownbook />

==Regulation of transcription in cancer==
{{main|Regulation of transcription in cancer}}

In vertebrates, the majority of gene [[啟動子|啟動子]] contain a [[CpG位点|CpG位点]] with numerous [[CpG位点|CpG位点]]s.<ref name="pmid16432200">{{cite journal |vauthors=Saxonov S, Berg P, Brutlag DL |title=A genome-wide analysis of CpG dinucleotides in the human genome distinguishes two distinct classes of promoters |journal=Proc. Natl. Acad. Sci. U.S.A. |volume=103 |issue=5 |pages=1412–7 |year=2006 |pmid=16432200 |pmc=1345710 |doi=10.1073/pnas.0510310103 |url=}}</ref>  When many of a gene's promoter CpG sites are [[DNA甲基化|DNA甲基化]] the gene becomes silenced.<ref name=Bird>{{cite journal |vauthors=Bird A |title=DNA methylation patterns and epigenetic memory |journal=Genes Dev. |volume=16 |issue=1 |pages=6–21 |year=2002 |pmid=11782440 |doi=10.1101/gad.947102 |url=}}</ref>  Colorectal cancers typically have 3 to 6 {{tsl|en|Somatic evolution in cancer#Glossary}} mutations and 33 to 66 {{tsl|en|Genetic hitchhiking}} or passenger mutations.<ref name="pmid23539594">{{cite journal |vauthors=Vogelstein B, Papadopoulos N, Velculescu VE, Zhou S, Diaz LA, Kinzler KW |title=Cancer genome landscapes |journal=Science |volume=339 |issue=6127 |pages=1546–58 |year=2013 |pmid=23539594 |pmc=3749880 |doi=10.1126/science.1235122 |url=}}</ref>   However, transcriptional silencing may be of more importance than mutation in causing progression to cancer.  For example, in colorectal cancers about 600 to 800 genes are transcriptionally silenced by CpG island methylation (see {{tsl|en|regulation of transcription in cancer}}).  Transcriptional repression in cancer can also occur by other {{tsl|en|Cancer epigenetics}} mechanisms, such as altered expression of [[小分子核糖核酸|小分子核糖核酸]].<ref name="pmid24616890">{{cite journal |vauthors=Tessitore A, Cicciarelli G, Del Vecchio F, Gaggiano A, Verzella D, Fischietti M, Vecchiotti D, Capece D, Zazzeroni F, Alesse E |title=MicroRNAs in the DNA Damage/Repair Network and Cancer |journal=Int J Genom |volume=2014 |issue= |pages=820248 |year=2014 |pmid=24616890 |pmc=3926391 |doi=10.1155/2014/820248 |url=}}</ref>  In breast cancer, transcriptional repression of {{tsl|en|BRCA1}} may occur more frequently by over-expressed microRNA-182 than by hypermethylation of the BRCA1 promoter (see {{tsl|en|BRCA1#Low expression of BRCA1 in breast and ovarian cancers}}).
-->
==真核转录控制==
真核生物中{{le|基因表达的调节|Regulation of gene expression}}是通过几种控制水平的相互作用实现的，这些控制在局部起作用以响应特定的细胞需要打开或关闭单个基因，并在全局范围内维持[[染色质|染色质]]范围的基因表达模式，从而塑造细胞身份<ref name=Watson /><ref>{{cite journal|last=Shandilya|first=J|author2=Robert SG|title=The transcription cycle in eukaryotes: from productive initiation to RNA polymerase II recycling|journal=Biochim Biophys Acta|year=2012|volume=1819|issue=5|pages=391–400|doi=10.1016/j.bbagrm.2012.01.010|pmid=22306664}}</ref>。 由于真核基因组包裹在组蛋白周围形成[[核小体|核小体]]和高阶染色质结构，转录机制的底物通常被部分隐藏<ref name=Watson />。 没有调节蛋白，许多基因以低水平表达或根本不表达。 转录需要置换定位的[[核小体|核小体]]，以使转录机制能够获得DNA<ref>{{cite journal|last=Kulaeva|first=Olga|author2=Daria Gaykalova |author3=Vasily M. Studitsky |title=Transcription Through Chromatin by RNA polymerase II: Histone Displacement and Exchange|journal=Mutat. Res.|year=2007|volume=618|issue=1–2|pages=116–129|doi=10.1016/j.mrfmmm.2006.05.040|pmc=1924643|pmid=17313961}}</ref>。

转录中的所有步骤都受到一定程度的调节<ref name=Watson />。 特别是转录起始是调节基因表达的主要水平。 针对细胞的能量成本而言，针对限速初始步骤是最有效的。 转录起始受DNA调节区内[[顺式作用元件|顺式作用元件]]（[[增强子|增强子]]，[[沉默子|沉默子]]，隔离子）和作为激活剂或阻遏物的序列特异性[[反式作用因子|反式作用因子]]的调节<ref name=Watson />。 通过靶向延长聚合酶的运动，也可以在起始后调节基因转录<ref name=Peterlin>{{cite journal|last=Peterlin|first=BM|author2=DH Price|title=Controlling the elongation phase of transcription with P-TEFb|journal=Molecular Cell|year=2006|volume=23|issue=3|pages=297–305|doi=10.1016/j.molcel.2006.06.014|pmid=16885020}}</ref>。

===全局控制和表观遗传调控===
{{main|表观遗传学}}
真核基因组被组织成紧凑的染色质结构，只允许对DNA进行调节。 染色质结构可以是全局“开放的”和更多转录允许的，或全局“浓缩的”和转录失活的。 前者（[[真染色質|真染色質]]）轻度包装(lightly packed)，在活跃转录下富含基因。 后者（[[异染色质|异染色质]]）包括基因贫乏区域，例如[[端粒|端粒]]和[[着丝粒|着丝粒]]，但也包括具有正常基因密度但转录沉默的区域。 转录可以通过[[組織蛋白#染色质调控|组蛋白修饰]]（[[组蛋白脱乙酰酶|脱乙酰化]]和{{le|组蛋白甲基化|Histone methylation|甲基化}}），[[RNA干扰|RNA干扰]]和/或[[DNA甲基化|DNA甲基化]]来沉默。

定义细胞身份的基因表达模式是通过[[细胞分裂|细胞分裂]]遗传的。 这个过程称为[[表观遗传学|表观遗传调控]]<ref name=Watson />。 DNA甲基化通过维持甲基化酶的作用可靠地遗传，甲基化酶修饰由复制产生的新生DNA链<ref name=Watson />。 在哺乳动物细胞中，DNA甲基化是转录沉默区域的主要标记。 专门的蛋白质可识别标记并募集[[组蛋白脱乙酰酶|组蛋白脱乙酰酶]]和甲基化酶以重建沉默。 核小体组蛋白修饰也可以在细胞分裂过程中遗传，但是，目前尚不清楚它是否能在没有DNA甲基化指导的情况下独立发挥作用<ref name=Watson />。

==参阅==
{{portal|分子与细胞生物学}}
* {{le|细菌转录|Bacterial transcription}}
* [[转录调控|转录调控]]
* [[转录因子|转录因子]]
* [[RNA聚合酶|RNA聚合酶]]
* {{link-en|基因表达调控|Regulation of gene expression}}

==参考文献==
{{reflist|30em}}

{{基因表达}}
{{转录}}

[[Category:基因表現|Category:基因表現]]
[[Category:分子生物学|Category:分子生物学]]